app/controllers/course/assessment/question/controller.rb

```diff
@@ -26,4 +32,15 @@ def load_question_assessment_for(question)
   def authorize_assessment
     authorize!(:update, @assessment)
   end
+
+  private
+
+  def assign_skills_to_question_assessment(question_assessment, skill_ids)
```

Comment:
Any issues with doing the assignment in this manner: `question_assessment.skill_ids = skill_ids`? Activerecord provides this as an API for associations.